app/javascript/components/time_table/timesheet/entry.js

```diff
@@ -11,6 +11,8 @@ class Entry extends React.Component {
   constructor (props) {
     super(props);
 
+    this.lastProject = null;
```

Comment:
why this value is inside of contstuctor, not in state / props?

Reply:
Because it is not used for rendering.